packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -620,6 +666,12 @@ function useMarkInboxNotificationAsRead_withClient(client: OpaqueClient) {
             readAt,
             optimisticId
           );
+
+          // Force a re-fetch of the unread notifications count
+          for (const poller of unreadNotificationsCountPollersByQueryKey.values()) {
+            poller.markAsStale();
+            poller.pollNowIfStale();
+          }
```

Comment:
I guess we can't really avoid doing a full refetch in these cases, so it's okay.